.env.example

```diff
@@ -17,8 +17,15 @@ GC_NOTIFY_CALLBACK_BEARER_TOKEN=
 # Used when running against LocalStack
 LOCAL_AWS_ENDPOINT=http://localhost:4566
 
+# Local dev 
+# Temporary token templateID
+TEMPORAY_TOKEN_TEMPLATE_ID=b6885d06-d10a-422a-973f-05e274d9aa86
+
```

Comment:
Are these the actual Notify values?
Although not exactly secret values they should probably not be listed in our `.env.example` file.  Always use fake strings when providing examples.

Reply:
I'm also aware these were hardcoded before but if we're going with a dynamic value it might be best to not use the real one in case it causes confusion.